services/provider/server/server.go

```diff
@@ -149,45 +170,22 @@ func (s *OCSProviderServer) OnboardConsumer(ctx context.Context, req *pb.Onboard
 		return nil, status.Errorf(codes.InvalidArgument, "onboarding ticket storageCluster not match existing storageCluster.")
 	}
 
-	storageQuotaInGiB := ptr.Deref(onboardingTicket.StorageQuotaInGiB, 0)
-
 	if onboardingTicket.SubjectRole != services.ClientRole {
 		err := fmt.Errorf("invalid onboarding ticket for %q, expecting role %s found role %s", req.ConsumerName, services.ClientRole, onboardingTicket.SubjectRole)
 		klog.Error(err)
 		return nil, status.Error(codes.InvalidArgument, err.Error())
 	}
 
-	storageConsumerUUID, err := s.consumerManager.Create(ctx, req, int(storageQuotaInGiB))
+	storageConsumerUUID, err := s.consumerManager.OnboardClient(ctx, storageConsumer)
```

Comment:
???
```suggestion
	storageConsumerUUID, err := s.consumerManager.EnableStorageConsumer(ctx, storageConsumer)
```

Reply:
my mistake I was fixated w/ OnboardClient and total forgot EnableStorageConsumer.

Reply:
fixed.